README.md

```diff
@@ -1,36 +1,63 @@
 # Tic Tac Toe Ruby Game (The Odin Project)
 
-> This project is part of the Microverse Ruby Curriculum and The Oding Project. 
+> This is a Tic Tac Toe game made with Ruby. It's a part of the Microverse Ruby Curriculum and The Odin Project. 
 
-## Built With
+## Built using
 
 - Ruby
+- Gitflow Workflow
+
+## UML Class diagram
+
+![screenshot](./UML_Class_diagram.png)
+
+## Project requirements
+- All user interaction and game flow should be kept only in main.rb
+- Use OOP
 
 ## Live Demo
-[Live preview:](https://repl.it/join/gjrzjjbm-marcoshdezcam1)
+[Live preview](https://gitpod.io/github.com/marcoshdezcam/Tic_Tac_Toe_Game/tree/feature_readme_game_instructions)
 
 ## Getting Started
 
-To run the game just open it with an IRB console or just run it. 
+### How to run the game?
+1 -> Clone this repo and open a terminal inside the folder where you extracted it.
+
+2 -> Run "Run `./bin/main.rb`" in your terminal
```

Comment:
- [ ] **OPTIONAL** just a typo, you have run run here

Reply:
Thanks!